base/tps/src/org/dogtagpki/server/tps/processor/TPSEnrollProcessor.java

```diff
@@ -302,6 +325,13 @@ private void enroll() throws TPSException, IOException {
             }
         }
 
+        //RedHat do this to check the fact that DES has been configured for the non external Reg legacy key wrapping.
+        if(!isExternalReg) {
+            String aesKeyWrapAlg =  getAESKeyWrapAlgSSKeyGen();
+            //We don't care about the anwer here, we just want to set the fact that des is configured.
```

Comment:
uh, it initially appeared to me that these few lines did nothing but printing out a debug message, so I had to look into getAESKeyWrapAlgSSKeyGen() and there I found that  it has been modified to call setSelectedKeyWrapAlg(...), which practically making the method getAESKeyWrapAlgSSKeyGen() a misnomer.  I'll put my comment at getAESKeyWrapAlgSSKeyGen() although I know that area is part of "external code contributions" which I'm not supposed to be reviewing so closely at.